ui/base-tool/src/main/java/org/cru/godtools/base/tool/activity/BaseToolActivity.java

```diff
@@ -306,8 +309,18 @@ protected final void stopDownloadProgressListener() {
 
     // endregion DownloadProgress logic
 
-    protected final void trackToolView(@NonNull final String tool) {
-        EventBus.getDefault().post(new ToolUsedEvent(tool));
+    protected final void trackToolOpen(@NonNull final String tool) {
+        EventBus eventBus = EventBus.getDefault();
```

Comment:
`mEventBus` is available from the `BaseActivity`

Reply:
I actually noticed the cleanup in the method earlier today, but didn't push it upstream yet